test/libsolidity/syntaxTests/memberLookup/memory_structs_with_mappings_unused.sol

```diff
@@ -0,0 +1,10 @@
+contract Test {
+	struct S { uint8 a; mapping(uint => uint) b; uint8 c; }
+	S s;
+	function f() public {
+		S memory x;
+		x.a;
+	}
+}
+// ----
+// TypeError: (104-114): Type struct Test.S memory is only valid in storage.
```

Comment:
This is the test of the breaking change that would fix https://github.com/ethereum/solidity/issues/6444.